api/client/src/main/java/org/projectnessie/client/auth/oauth2/OAuth2Client.java

```diff
@@ -250,10 +255,14 @@ Tokens fetchNewTokens() {
       case AUTHORIZATION_CODE:
         try (AuthorizationCodeFlow flow = new AuthorizationCodeFlow(config)) {
           return flow.fetchNewTokens();
+        } finally {
+          lastAccess = config.getClock().get();
```

Comment:
nit: Does it make sense to _also_ set `lastAccess` before we start `AuthorizationCodeFlow` in case the flow runs for a long time?